src/systems/filecoin_mining/storage_mining/storage_miner_actor.id

```diff
@@ -144,7 +144,7 @@ type StorageMinerActorCode struct {
         faultType      sector.StorageFaultType
     )
 
-    _submitPowerReport(rt Runtime, lastPoSt block.ChainEpoch)
+    _submitPowerReport(rt Runtime, lastPoSt block.ChainEpoch, getActiveDealIDs bool)
```

Comment:
The argument is actually called `processDealPayment`, which is very different!

Split this into two methods, this and `claimDealPayments`, the second of which is only called after successful PoSt.